docs/simnet_environment.mediawiki

```diff
@@ -0,0 +1,321 @@
+*** Automatically votes
+* Helper scripts accessible from the lower pane in each window that communicate with the associated process in the upper pane
+* Scripts available in the directories of all instances:
+** <code>ctl</code> - Invokes <code>dcrctl</code> communicating to the associated instance process
+** <code>stopall</code> - Stops all of the processes and closes the <code>tmux</code> session
+* Additional scripts for the <code>dcrd1</code> and <code>dcrd2</code> instances:
+** <code>mine</code> - Mines a specified number of blocks with a delay between
+* Additional scripts for the <code>dcrd2</code> instance:
+** <code>reorg</code> - Forces a chain reorganization in the instance
+* Additional scripts for the <code>wallet1</code> and <code>wallet2</code> instances:
+** <code>tickets</code> - Manually purchases the specified number of tickets with a default of 1
+** <code>xfer</code> - Creates a transaction that transfers (sends) funds to an address owned by the instance
+
+===2.2. Using the Preconfigured Environment===
+
+In order to get started, first ensure that the <code>dcrd</code>,
+<code>dcrwallet</code>, and <code>dcrctl</code> binaries you want to test with
+are available in your system path:
+
+  $ dcrd -V
+  dcrd version x.y.z (Go version <version> <os>)
+  $ dcrwallet -V
+  dcrwallet version x.y.z (Go version <version> <os>)
+  $ dcrctl -V
+  dcrctl version x.y.z (Go version <version> <os>)
+
+Once that above has been verified, run the setup script:
+
+  $ ./dcr_tmux_simnet_setup.sh
+
+It will take a few seconds before the environment appears since it mines a bunch
+of initial blocks so that the environment has spendable coins immediately
+available.
+
+Once the environment launches, you will be in the bottom pane of the
+<code>dcrd1</code> window.  As the overview section describes, this lower pane
+provides convenience scripts for interacting with the <code>dcrd</code> instance
+running in the upper pane.
+
+At this point, everything is setup and ready to go.  Since it is running in a
+[https://github.com/tmux/tmux tmux] session, the various windows and panes can
+be accessed with normal <code>tmux</code> command sequences.  By default,
+<code>Ctrl+B</code> followed by a window number will switch to that number.  For
+example <code>Ctrl+B 1</code> will switch the window with the
+<code>dcrwallet1</code> instance and <code>Ctrl+B 0</code> will switch back to
+the window with the <code>dcrd1</code> instance.
+
+See the next sections for a quick tour of performing some basic tasks.
+
+====2.2.1 Basic Tasks - Interfacing with the <code>dcrd1</code> instance====
+
+Ensure the window that houses the <code>dcrd1</code> instance is active and the
+lower pane has focus.  This '''might''' require switching to the window with the
+tmux command sequence <code>Ctrl+B 0</code> and <code>Ctrl+B <down arrow></code>
+to switch to the pane.
+
+* Query the current best block of the private network:
+  $ ./ctl getblockcount
```

Comment:
This only returns a number for me (on master) . https://github.com/decred/dcrd/blob/master/docs/json_rpc_api.mediawiki#getblockcount  also indicates the same.

Reply:
Indeed.  This was supposed to say `./ctl getbestblock`.  Updated.